src/Azure.Functions.Cli/Kubernetes/KEDA/KedaHelper.cs

```diff
@@ -52,10 +44,6 @@ public static async Task<KedaVersion> DetermineCurrentVersion(string @namespace)
                 // We use KEDA v2 as a default
                 return KedaVersion.v2;
             }
```

Comment:
Please revers this, it should still be added in this PR

```suggestion
            }
            
            var parsedJson = JToken.Parse(resourceInfo.Output);	
            // TODO: Interpret app.kubernetes.io/version label	

```